src/device/riscv/riscv.go

```diff
@@ -10,10 +10,10 @@ func Asm(asm string)
 // recognizes template values in the form {name}, like so:
 //
 //	arm.AsmFull(
-//	    "st {value}, {result}",
+//	    "str {value}, [{result}]",
```

Comment:
`str` is not a RISC-V instruction.
```suggestion
//	    "st {value}, [{result}]",
```